### ISSUE TITLE
running a single test is much faster, running the entire suite much slower?

### ISSUE THREAD

**Issue by Fjan** [rails/spring#565]
Adding the spring gem to a rails project improves the time to run a single test owing to faster start-up speed as expected. However, the total test time increases by almost 70%:

without spring `time rake test`
```
real	0m29.822s
user	0m24.935s
sys	0m2.086s
```
with spring `time bin/rake test`
```
real	0m49.423s
user	0m0.172s
sys	0m0.044s
```

Is anybody else seeing this? Any guidance on how to further debug this issue would be appreciated.

**Comment by rafaelfranca** [comment]
Are you on mac os? If so that is a bug in Ruby

**Comment by Fjan** [comment]
Yes, I'm on Mac OS. I've tried it with both Ruby 2.4 and 2.5 and confirmed with two other developers on the same project that they're seeing this. Is this in the Ruby bug tracker somewhere?

**Comment by Fjan** [comment]
Thanks a lot for the explanation! I can't believe Google / Stack Overflow failed me on this one. Hopefully this issue will et indexed properly.
